src/constants/validation/files.js

```diff
@@ -0,0 +1,17 @@
+export const certificates = (files) => {
```

Comment:
This is not a constant. It is a helper (utils). And it should be general just receiving file size in MBytes.

Reply:
ok